### PR TITLE
f-registration@v3.9.0 - Update visual tests to include multiple tenants

### PR DESCRIPTION
**Pull request by siggerzz**
v3.9.0
------------------------------
*October 28, 2022*

### Changed
- Visual tests to include new tenants
- Updated Story file to use Storybook `Controls`.

### CODE REVIEW

File: packages/components/pages/f-registration/stories/Registration.stories.js [hunk 1]
```diff
@@ -50,4 +28,38 @@ export const RegistrationComponent = () => ({
...
+    },
+    title : {
+        control: { type: 'text' },
+        description: 'Title Text',
+        defaultValue: 'Create Account'
+    },
+    createAccountUrl : {
+        control: { type: 'text' },
+        description: 'Create Account URL',
+        defaultValue: '/account/login'
```
Comment (adammorr) [inline packages/components/pages/f-registration/stories/Registration.stories.js]:
```suggestion
        defaultValue: '/account/register'
```

File: packages/components/pages/f-registration/test/visual/f-registration.visual.desktop.spec.js [hunk 2]
```diff
@@ -5,79 +5,83 @@ const devices = [
...
-
-            // Act
-            Object.keys(userInfo).forEach(field => Registration.setFieldValue(field, userInfo[field]));
-            await browser.pause(500);
-            await Registration.submit();
-
-            // Assert
-            await browser.percyScreenshot('f-registration - "Email is already registered error"', device);
-        });
+const locales = ['en-GB', 'en-AU', 'en-NZ', 'es-ES', 'it-IT'];
```
Comment (adammorr) [inline packages/components/pages/f-registration/test/visual/f-registration.visual.desktop.spec.js]: Is this meant to be missing `en-IE`?

**Review by adammorr — CHANGES_REQUESTED** [review]

File: packages/components/pages/f-registration/test/visual/f-registration.visual.desktop.spec.js (hunk 2 above)
Comment (siggerzz) [reply packages/components/pages/f-registration/test/visual/f-registration.visual.desktop.spec.js]: Ah yeah, my bad. Bad copy paste job 😁

**Comment by siggerzz** [comment]
@adammorr I took the opportunity to remove `.desktop` from the visual test file name, hence why it looks like the whole file has changed.

This file name used to determine whether or not to run Percy checks on mobile / desktop, but it looks like we missed this when refactoring to use the `devices` array.

**Review by adammorr — DISMISSED** [review]

**Review by philgoff — APPROVED** [review]

**Review by adammorr — APPROVED** [review]

**Pull request merged**